double_entry.gemspec

```diff
@@ -17,6 +17,32 @@ Gem::Specification.new do |gem|
   gem.test_files    = gem.files.grep(%r{^(test|spec|features)/})
   gem.require_paths = ['lib']
 
+  gem.post_install_message = <<-'POSTINSTALLMESSAGE'
+Please note the following changes in DoubleEntry:
+ - New table `double_entry_line_metadata` has been introduced and is required for
```

Comment:
yeah i think it would be nicer as well, but, when you try to access it this is what happens:
```
There was a NoMethodError while loading double_entry.gemspec:
undefined method `table_name_prefix' for DoubleEntry:Module from
  /Users/asellitt/src/double_entry/double_entry.gemspec:28:in `block in <main>'
```
i think its ok to mention that there is a config, i wouldnt expect that the sample migration would be able to be cleanly copied and pasted, so i think that reminding them of the config they may/may not have set up is a good idea